tests/IoTeX/StakingTests.cpp

```diff
@@ -132,22 +132,22 @@ TEST(TWIoTeXStaking, CandidateUpdate) {
 }
 
 TEST(TWIoTeXStaking, SignAll) {
-    auto input = Proto::SigningInput();
-    input.set_version(1);
-    input.set_nonce(0);
-    input.set_gaslimit(1000000);
-    input.set_gasprice("10");
     auto keyhex = parse_hex("cfa6ef757dee2e50351620dca002d32b9c090cfda55fb81f37f1d26b273743f1");
-    input.set_privatekey(keyhex.data(), keyhex.size());
-    Proto::SigningOutput output;
 
     { // sign stakecreate
-        auto action = input.mutable_stakecreate();
-        action->set_candidatename("io19d0p3ah4g8ww9d7kcxfq87yxe7fnr8rpth5shj");
-        action->set_stakedamount("100");
-        action->set_stakedduration(10000);
-        action->set_autostake(true);
-        action->set_payload("payload");
+        auto input = Proto::SigningInput();
+        input.set_version(1);
+        input.set_nonce(0);
+        input.set_gaslimit(1000000);
+        input.set_gasprice("10");
+        input.set_privatekey(keyhex.data(), keyhex.size());
+        Proto::SigningOutput output;
```

Comment:
move to a `func` for this duplicated input code

Reply:
+1; done